src/Npgsql/NpgsqlParameterCollection.cs

```diff
@@ -452,6 +466,7 @@ public void Remove(string parameterName)
     /// Removes the specified <see cref="NpgsqlParameter"/> from the collection.
     /// </summary>
     /// <param name="value">The <see cref="NpgsqlParameter"/> to remove from the collection.</param>
+
```

Comment:
Please undo this newline. You probably wanted to replace this value doc with the same remarks as on Insert etc.

Reply:
Mind adding your version of the doc to this Remove method too? That's all the feedback from me!

Reply:
It's my bad, there is a Remove(object value) overload below this method that I intended to comment on. Please take a look, it doesn't have the remarks yet.